planner/core/memtable_predicate_extractor.go

```diff
@@ -828,7 +828,13 @@ func (e *InspectionResultTableExtractor) Extract(
 }
 
 func (e *InspectionResultTableExtractor) explainInfo(p *PhysicalMemTable) string {
-	return ""
+	if e.SkipInspection {
+		return "skip_inspection: true"
```

Comment:
```suggestion
		return "skip_inspection:true"
```

to be compatible with others